samples/tab-device-permissions/nodejs/src/components/Tab.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Flex } from '@fluentui/react-northstar'
+import ScanBarCode from './ScanBarCode';
+import GetGeoLocation from './GetGeoLocation';
+import CaptureImage from './CaptureImage';
+import PeoplePicker from './PeoplePicker';
+import CaptureAudio from './CaptureAudio';
+
+/**
+ * The 'Tab' contains all the components
+ * of your app.
+ */
+const Tab = () => {
+    return(
+      <Flex column gap="gap.large">
+       {/* Component to capture image(s) */}
```

Comment:
Can we render these in a grid like view? Current view looks fine only on mobile.